tests/providers/microsoft/azure/hooks/test_wasb.py

```diff
@@ -52,70 +52,82 @@ def setup(self):
         self.public_read_conn_id = 'pub_read_id'
         self.managed_identity_conn_id = 'managed_identity'
 
+        self.proxies = {'http': 'http_proxy_uri', 'https': 'https_proxy_uri'}
+
         db.merge_conn(
             Connection(
                 conn_id=self.public_read_conn_id,
                 conn_type=self.connection_type,
                 host='https://accountname.blob.core.windows.net',
+                extra=json.dumps({'proxies': self.proxies}),
             )
         )
 
         db.merge_conn(
             Connection(
                 conn_id=self.connection_string_id,
                 conn_type=self.connection_type,
-                extra=json.dumps({'connection_string': CONN_STRING}),
+                extra=json.dumps({'connection_string': CONN_STRING, 'proxies': self.proxies}),
             )
         )
         db.merge_conn(
             Connection(
                 conn_id=self.shared_key_conn_id,
                 conn_type=self.connection_type,
                 host='https://accountname.blob.core.windows.net',
-                extra=json.dumps({'shared_access_key': 'token'}),
+                extra=json.dumps({'shared_access_key': 'token', 'proxies': self.proxies}),
             )
         )
         db.merge_conn(
             Connection(
                 conn_id=self.ad_conn_id,
                 conn_type=self.connection_type,
-                extra=json.dumps(
-                    {'tenant_id': 'token', 'application_id': 'appID', 'application_secret': "appsecret"}
-                ),
+                host='conn_host',
+                login='appID',
+                password='appsecret',
+                extra=json.dumps({'tenant_id': 'token', 'proxies': self.proxies}),
```

Comment:
This connection was not used in tests and had some probles.

Reply:
Looks like it's worth fixing(in a separate PR though)